packages/core/src/components/modal/test/default/modal.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/modal/test/default/index.html';
+
+test.describe('tds-modal-default', () => {
+  test('renders modal correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tdsModal = page.getByTestId('tds-modal-testid');
+    await expect(tdsModal).toHaveCount(1);
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('modal contains two buttons', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tdsModal = page.getByTestId('tds-modal-testid');
+
```

Comment:
[Unit test scenario](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=64N7Zh&nav=MTVfezIyRjA3ODQ0LUIwRkItNDIzNS05MDk4LTIwOUM4ODU1Q0MwNn0) suggests testing the Header and Body test too, can that be added too?